Sources/Controllers/2.0/OATargetInfoViewController.mm

```diff
@@ -465,7 +470,24 @@ - (UITableViewCell *) tableView:(UITableView *)tableView cellForRowAtIndexPath:(
     
     if (!info.isHtml)
     {
-        if (info.collapsable)
+        if (info.isCoordinates)
+        {
+            OATargetInfoCollapsableCoordinatesViewCell *cell;
+            cell = (OATargetInfoCollapsableCoordinatesViewCell *)[tableView dequeueReusableCellWithIdentifier:reusableIdentifierCollapsable];
+            
+            if (cell == nil || ![cell isKindOfClass:[OATargetInfoCollapsableCoordinatesViewCell class]])
```

Comment:
I'm not shore about this. But without it I have Unrecognized selector  error. Because instead [OATargetInfoCollapsable**Coordinates**ViewCell setupCellWithLatLon] it called  it parent class without this method [OATargetInfoCollapsableViewCell setupCellWithLatLon]. I tried to cast that code, but it didn't help:  [((OATargetInfoCollapsableCoordinatesViewCell *)cell) setupCellWithLat:info.lat lon:info.lon];